ravem/indico_ravem/util.py

```diff
@@ -112,16 +107,28 @@ def has_access(event_vc_room, _split_re=re.compile(r'[\s,;]+')):
     event = event_vc_room.event
     current_user = session.user
 
-    # No physical room or room is not Vidyo capable
-    if not room or not room.has_equipment('Vidyo'):
+    # No physical room or room is not videoconference capable
+    feature = RavemPlugin.settings.get('room_feature')
+    equipment = EquipmentType.query.filter(EquipmentType.features.any(RoomFeature.name == feature)).all()
+    if not room or not room.has_equipment(*(e.name for e in equipment)):
         return False
 
-    ips = set(filter(None, (x.strip() for x in _split_re.split(room.get_attribute_value('ip', '')))))
-    return any([
-        current_user == retrieve_principal(vc_room.data.get('owner')),
-        event.can_manage(current_user),
-        request.remote_addr in ips
-    ])
+    ips = set(
+        filter(
+            None,
```

Comment:
Please don't add filter-None in new code, this gets rather ugly when converting to Python 3 since it no longer exists there.

Reply:
ah this looks like a black reformatting... so let's keep the logic and just undo the formatting. this will otherwise cause merge conflicts with master there where it has been rewritten to this by *modernize*:

    ips = {_f for _f in (x.strip() for x in _split_re.split(room.get_attribute_value('ip', ''))) if _f}